Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Annotation/AddIshAnnotation.cs

```diff
@@ -0,0 +1,365 @@
+﻿/*
+* Copyright (c) 2014 All Rights Reserved by the SDL Group.
+* 
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+* 
+*     http://www.apache.org/licenses/LICENSE-2.0
+* 
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Trisoft.ISHRemote.Objects;
+using Trisoft.ISHRemote.Objects.Public;
+using Trisoft.ISHRemote.Exceptions;
+using Trisoft.ISHRemote.DocumentObj25ServiceReference;
+
+namespace Trisoft.ISHRemote.Cmdlets.Annotation
+{
+    /// <summary>
+    /// <para type="synopsis">The Add-IshAnnotation cmdlet adds a new annotation to the specified content object</para>
+    /// <para type="description">The Add-IshAnnotation cmdlet adds a new annotation to the specified content object</para>
+    /// </summary>
+    /// <example>
+    /// <code>
+    /// $ishSession = New-IshSession -WsBaseUrl "https://example.com/InfoShareWS/" -IshUserName "username" -IshUserPassword  "userpassword"
+    /// $metadata = Set-IshMetadataField -Name "FISHREVISIONID" -Level Annotation -Value "GUID-MYREVISIONID" | `
```

Comment:
Please remove all ` (back-tick) from examples... an enter is allowed behind a | (pipe) character, only use the back-tick when separating parameters for readability... the back-tick is error prone as it only escapes the character right after it so "back-tick space enter" still gives an error

Reply:
Done